Makefile

```diff
@@ -1,41 +1,56 @@
-install-root-dependencies:
+help:                                                                           ## shows this help
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_\-\.]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
```

Comment:
I needed a quick reference of what's in here, so I added a default `help` command that prints a list of what's going on.